delta-api/src/main/java/io/cdap/delta/api/DDLEvent.java

```diff
@@ -30,15 +30,13 @@
 public class DDLEvent extends ChangeEvent {
   private final DDLOperation operation;
   private final Schema schema;
-  private final String database;
   private final List<String> primaryKey;
 
-  private DDLEvent(Offset offset, String database, DDLOperation operation, @Nullable Schema schema,
+  private DDLEvent(Offset offset, DDLOperation operation, @Nullable Schema schema,
```

Comment:
schema is not required in this class since it will come from DDLOperation.

Reply:
This is table schema, not schema name. Do we want to move the table schema to DDLOpration too ?

Reply:
ah its ok then. thanks for clarification.